test/extended/user_tags/azure.go

```diff
@@ -0,0 +1,75 @@
+package usertags
+
+import (
+	"context"
+	"fmt"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	"github.com/stretchr/objx"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	azutil "github.com/openshift/origin/test/extended/util/azure"
+)
+
+var _ = g.Describe("[sig-installer][Feature:AzureUserTags] Managed cluster should", func() {
+	defer g.GinkgoRecover()
+
+	g.It("have user defined tags present on all the created resources", func() {
```

Comment:
need `[apigroup:config.openshift.io]` added here to avoid failures on microshift

Reply:
incorporated comment.